pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -1016,7 +1016,11 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
 
         // Does the player own the airport?
         if (endid != playerid)
-            return 0;
+        {
+            SendClientMessage(playerid, COLOR_ORANGE, "Error: Sorry, You Don't Own The LV Airport"); //Fix "Server: Unkown Command" issue
```

Comment:
Please Don't Begin Each Word With A Capital.

That's not how the English language works. We require proper English grammer in the messages shown on Las Venturas Playground. What about:

> Error: Sorry, you can only use this command when you own the LV Airport!